atmcfg/atmcfg_test.go

```diff
@@ -140,6 +171,15 @@ func TestAddUser(t *testing.T) {
 	}
 }
 
+func TestAddIndexConfig(t *testing.T) {
+	config := automationConfigWithIndexConfig()
```

Comment:
[q] can you add a scenario for an empty  IndexConfigs? I want to make sure we won't hit a null pointer error here